docs/icp/17-box-shadow-hover/17-box-shadow-hover.md

```diff
@@ -133,7 +135,276 @@ box-shadow: 5px 5px 5px 2px #808080;
 
 <img src="/icp/17/step-2.png" alt="step-2" width="500px"/>
 
-
 In this example, there is an image element with the class `img-tiger`. The CSS styles defined for the class `img-tiger` specify that the image should have a height of `300px`, a border-radius of `50px`, and a margin of `30px`. The `box-shadow` property is used to add a shadow to the image. The values `2px 2px 5px 10px` define the horizontal offset, vertical offset, blur radius, and spread radius of the shadow, respectively. The color of the shadow is specified as `rgb(138, 136, 136)`.
 
-When you view this HTML page in a browser, the image will have a box shadow applied to it according to the styles defined by the `.img-tiger` class. When you move your mouse over the image, the border-radius will change to 20px with a transition effect, as defined by the `.img-tiger:hover` selector.
\ No newline at end of file
+When you view this HTML page in a browser, the image will have a box shadow applied to it according to the styles defined by the `.img-tiger` class. When you move your mouse over the image, the border-radius will change to 20px with a transition effect, as defined by the `.img-tiger:hover` selector.
+
+## Background image
+
+The `background-image` property is used to set one or more background images.
```

Comment:
The background-image property in CSS is used to set one or more background images for an element.